src/sdp/DefaultSDP.ts

```diff
@@ -218,41 +218,75 @@ export default class DefaultSDP implements SDP {
       )
     );
     const srcLines: string[] = this.lines();
-    const dstLines: string[] = [];
+    const fmtpAttribute = DefaultSDP.findOpusFmtpAttribute(srcLines);
+    const dstLines = DefaultSDP.updateOpusFmtpAttribute(srcLines, fmtpAttribute, [
+      `maxaveragebitrate=${maxAverageBitrate}`,
+    ]);
+    return DefaultSDP.linesToSDP(dstLines);
+  }
+
+  withStereoAudio(): DefaultSDP {
+    const srcLines: string[] = this.lines();
+    const fmtpAttribute = DefaultSDP.findOpusFmtpAttribute(srcLines);
+    const dstLines = DefaultSDP.updateOpusFmtpAttribute(srcLines, fmtpAttribute, [
+      'stereo=1',
+      'sprop-stereo=1',
+    ]);
+    return DefaultSDP.linesToSDP(dstLines);
+  }
+
+  static findOpusFmtpAttribute(sdpLines: string[]): string | null {
```

Comment:
SDP code is very hard to understand for me and other people in JS SDK. Can we put documentation on  both these static method and maybe more comment inside the method on which loop does what.

Reply:
Added some comments